heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -161,6 +162,25 @@ void killTopologyHandler(String topologyName) throws TopologyRuntimeManagementEx
     LOG.fine("Scheduler killed topology successfully.");
   }
 
+  boolean consolePrompt(int oldContainerCount, int newContainerCount) {
```

Comment:
maybe rename to something like confirmWithUser(). How do you think?

Reply:
updated